app/src/main/java/org/sopt/and/designsystem/component/appbar/OnboardingTopBar.kt

```diff
@@ -0,0 +1,65 @@
+package org.sopt.and.designsystem.component.appbar
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import org.sopt.and.designsystem.theme.ANDANDROIDTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun OnboardingTopBar(
+    modifier: Modifier = Modifier,
+    title: String = "",
+    onBackClick: () -> Unit = {},
+    showBackBtn: Boolean = false,
+) {
```

Comment:
```suggestion
fun OnboardingTopBar(
    modifier: Modifier = Modifier,
    title: String = "",
    showBackBtn: Boolean = false,
    onBackClick: () -> Unit = {},
) {
```
람다를 맨 밑으로 내리면 중괄호로 접근이 가능합니다.

```
OnboardingTopBar() {
    // 클릭했을 때의 내용을 넣을 수 있어요
}
```